tests/fixtures.py

```diff
@@ -89,10 +88,6 @@ def create_pod(request):
     """
     class_instance = request.node.cls
 
-    pod_data = templating.load_yaml_to_dict(constants.CSI_RBD_POD_YAML)
-    pod_data['metadata']['name'] = helpers.create_unique_resource_name(
-        'test', 'pod'
+    class_instance.pod_obj = helpers.create_pod(
+        interface_type=constants.CEPHBLOCKPOOL, pvc=class_instance.pvc_obj.name
```

Comment:
since it always creates pod with cephblockpool, can we change fixture name to something like create_pod_cephblockpool

Reply:
done